content/plugins/descriptions/jira.yaml

```diff
@@ -79,16 +79,16 @@ gettingStarted:
       );
   - intro: Add annotation to the yaml config file of a component
     language: yaml
-    code: | 
+    code: |
       metadata:
         annotations:
-          jira/project-key: [example-jira-project-key]
-          jira/component: [example-component] // optional, you might skip this value to fetch data for all components
+          jira/project-key: &lt;example-jira-project-key>
+          jira/component: &lt;example-component> # optional, you might skip this value to fetch data for all components
   - intro: Get and provide `JIRA_TOKEN` as env variable in following format
     language: yaml
     code: |
       "Basic base64(jira-mail@example.com:JIRA_TOKEN)"
-      e.g. `Basic: ZXhhbXBsZV9qaXJhQGV4YW1wbGUuY29tOjU1Q3NUSEoxWW1oTVdJSFptdGJXNUUxOA==`
+      e.g. `Basic ZXhhbXBsZV9qaXJhQGV4YW1wbGUuY29tOjU1Q3NUSEoxWW1oTVdJSFptdGJXNUUxOA==`
```

Comment:
I am not sure that I understand what I need to do in order to use this plugin after reading this. 

Can we simplify it or perhaps use the notes section of this page to show the user where to get this token and how to pass it to Backstage in an environment variable?

Reply:
I've got the proper text but don't know if I can format it properly in this template:
![image](https://user-images.githubusercontent.com/24685983/101693283-ae01c880-3a71-11eb-9cde-8825f98f4069.png)

Reply:
@mcalus3 The note is great and it is much easier to understand the instructions now. Two more small things are needed:

1. The installation steps should tell the user that they need to provide a `JIRA_TOKEN` environment variable and they should check the instructions at the bottom of the page to learn how to generate that. It's too easy to miss the note under the screenshot.
2. The note should show the user how to pass the environment variable to Backstage. i.e. `env JIRA_TOKEN=xyz yarn dev`

This is ready to go once those two small things are in place.

Reply:
Done